### DISCUSSION TITLE
Restart Button is broken

### DISCUSSION THREAD

**Discussion by Pokeylooted** [Guilouz/Creality-Helper-Script-Wiki#578] [Issues]
Hey! If you click reboot in mainsail/fluidd It doesn't properly restart. Im guessing this is a limitation in K1's mb, however a warning might be nice!

**Comment by Guilouz** [comment]
No it’s working: https://guilouz.github.io/Creality-Helper-Script-Wiki/helper-script/moonraker-k1/
